ui/src/notebooks/components/Notebook.tsx

```diff
@@ -3,13 +3,18 @@ import React, {FC} from 'react'
 
 // Components
 import {Page} from '@influxdata/clockface'
+import NotebookHeader from 'src/notebooks/components/header'
+import PipeList from 'src/notebooks/components/PipeList'
+import MiniMap from 'src/notebooks/components/minimap/MiniMap'
+
+// Contexts
 import {ResultsProvider} from 'src/notebooks/context/results'
 import {RefProvider} from 'src/notebooks/context/refs'
 import CurrentNotebook from 'src/notebooks/context/notebook.current'
 import {ScrollProvider} from 'src/notebooks/context/scroll'
-import NotebookHeader from 'src/notebooks/components/header'
-import PipeList from 'src/notebooks/components/PipeList'
-import MiniMap from 'src/notebooks/components/minimap/MiniMap'
+import QueryProvider from 'src/notebooks/context/query'
+import {TimeProvider} from 'src/notebooks/context/time'
+import AppSettingProvider from 'src/notebooks/context/app'
```

Comment:
why did these have to be moved here?

Reply:
I added some comments to the PR:

https://github.com/influxdata/influxdb/pull/19396#discussion_r474145903

Reply:
@drdelambre needed to bump this higher in the tree so that the bucket cell can consume query provider, and query provider needs the other 2